src/components/richLink.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { css } from '@emotion/core';
+import { from } from '@guardian/src-foundations/mq';
+import { Design, Display, Pillar } from '@guardian/types/Format';
+import { text, withKnobs } from '@storybook/addon-knobs';
+import type { FC } from 'react';
+import React from 'react';
+import { basePx } from 'styles';
+import { selectPillar } from '../storybookHelpers';
+import RichLink, { richLinkWidth } from './richLink';
+
+const overrideStyle = css`
+	${from.wide} {
+		margin-left: calc(${richLinkWidth} + ${basePx(2)} + ${basePx(3)});
```

Comment:
nice, looks smart would like a walk through if poss 🎉

Reply:
The rich link component has a negative margin so it goes offscreen in storybook, so this is the opposite margin to bring it back on screen 😅